DependencyInjection/Configuration.php

```diff
@@ -78,8 +78,9 @@ public function getConfigTreeBuilder()
                 ->scalarNode('translation_domain')->defaultValue('messages')->end()
                 ->scalarNode('title')->end()
                 ->scalarNode('description')->end()
-                ->scalarNode('original_route_pattern')->defaultValue(SeoPresentation::ORIGINAL_URL_CANONICAL)->end()
-       ;
+                ->scalarNode('original_route_pattern')->defaultValue(SeoPresentation::ORIGINAL_URL_CANONICAL)
+            ->end()
```

Comment:
the `;` should be on a new line, but the end you moved down ends the original_route_pattern node. correct would be to add another end()